hopli/src/identity.rs

```diff
@@ -62,11 +79,16 @@ impl PrivateKeyArgs {
     }
 }
 
+/// Arguments for password.
```

Comment:
On longer descriptions, a line break is needed to separate the short description (first line) and detailed description (the one you get once you click on that function in type's function list)

```suggestion
/// Arguments for password.
///
```